src/ui/FileViewer/ImagesView.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+import { isImage } from '../../utils';
+import Label, { LabelTypography, LabelColors } from '../Label';
+
+const BUTTON_ICON_SIDE_LENGTH = '24px';
+
+export function ImagesView({ props, stringSet }: {
+  props: FileViewerComponentProps,
+  stringSet?: any,
+}) {
+  if (props.viewerType === ViewerTypes.MULTI) {
+    const { fileInfoList, currentIndex, onClickLeft, onClickRight } = props;
+    const { name, url, type } = fileInfoList[currentIndex];
+    return (
+      !isImage(type) || fileInfoList.length <= 1
+        ? <div className="sendbird-fileviewer__content">
+          <div
+            className="sendbird-fileviewer__content__unsupported"
+          >
+            <Label type={LabelTypography.H_1} color={LabelColors.ONBACKGROUND_1}>
+              {stringSet?.UI__FILE_VIEWER__UNSUPPORT || 'Unsupported message'}
+            </Label>
+          </div>
+        </div>
+        : <div className="sendbird-filesviewer-content">
+          <div className="sendbird-filesviewer-arrow left">
```

Comment:
`sendbird-filesviewer-arrow left ` -> `sendbird-filesviewer-arrow--left `

Reply:
The CSS naming are for parity with other components

Reply:
Is this correct? IDE cannot locate it if I change it.